assets/js/script.js

```diff
@@ -0,0 +1,548 @@
+  createPlatform('95px', '235px', $('#game1'), '25px', '7px', 'floor1')
+  createPlatform('143px', '176px', $('#game1'), '27px', '7px', 'floor1')
+
+  createPlatform('85px', '80px', $('#game2'), '25px', '7px', 'floor2')
+  createPlatform('0', '63px', $('#game2'), '16px', '100px', 'wall2')
+  createPlatform('95px', '150px', $('#game2'), '20px', '7px', 'floor2')
+  createPlatform('70px', '163px', $('#game2'), '20px', '7px', 'floor2')
+  createPlatform('55px', '195px', $('#game2'), '30px', '7px', 'floor2')
+  checkClear = setInterval(levelClear, 500)
+  return $('.nextLevel').remove()
+}
+
+function level5 () {
+  $('.goal').remove()
+  $('.platform').remove()
+
+  createGoal('0', '274px', $('#game1'))
+  createGoal('0', '0', $('#game2'))
+  start1.push($('.goal:first').position())
+  start2.push($('.goal:last').position())
+  currentLevel = $('.goal').last().attr('id') / 2
+  createPlatform('163px', '130px', $('#game1'), '70px', '7px', 'floor1')
+  createPlatform('50px', '150px', $('#game1'), '40px', '30px', 'wall1')
+  createPlatform('143px', '220px', $('#game1'), '30px', '7px', 'floor1')
+  createPlatform('100px', '130px', $('#game1'), '70px', '7px', 'floor1')
+  createPlatform('23px', '210px', $('#game1'), '80px', '7px', 'floor2')
+
+  createPlatform('95px', '195px', $('#game2'), '28px', '7px', 'floor2')
+  createPlatform('95px', '260px', $('#game2'), '40px', '7px', 'floor2')
+  createPlatform('0', '179px', $('#game2'), '16px', '100px', 'wall2')
+  createPlatform('135px', '250px', $('#game2'), '28px', '7px', 'floor2')
+  createPlatform('145px', '100px', $('#game2'), '130px', '7px', 'floor2')
+  createPlatform('99px', '0', $('#game2'), '70px', '7px', 'floor2')
+  createPlatform('49px', '60px', $('#game2'), '20px', '7px', 'floor2')
+  createPlatform('30px', '10px', $('#game2'), '30px', '7px', 'floor2')
+
+  checkClear = setInterval(levelClear, 500)
+  return $('.nextLevel').remove()
+}
+
+// to check if winning conditions have been met
+function levelClear () {
+  if (ball1Goal() === true && ball2Goal() === true) {
+    switch (currentLevel) {
+      case 1:
+        clearInterval(checkClear)
+        $('.container').append('<button class ="nextLevel" onclick="level2()">Next Level (N)</button>')
+        break
+      case 2:
+        clearInterval(checkClear)
+        $('.container').append('<button class ="nextLevel" onclick="level3()">Next Level (N)</button>')
+        break
+      case 3:
+        clearInterval(checkClear)
+        $('.container').append('<button class ="nextLevel" onclick="level4()">Next Level (N)</button>')
+        break
+      case 4:
+        clearInterval(checkClear)
+        $('.container').append('<button class ="nextLevel" onclick="level5()">Next Level (N)</button>')
+        break
+      case 5:
+        clearInterval(checkClear)
+        $('.container').append('<button class ="nextLevel" onclick="alert1()">Thank you for playing!</button>')
+        break
+    }
+  }
+}
```

Comment:
whew. what just happened

Reply:
this section of code represents when the user wins the current level. The logic which checks the winning condition is actually on a set interval, which also populates a nextLevel button. So in order to stop adding multiple buttons continously, we clear this interval, while adding a button which leads to the next level.